vendor/vendor-android/base/src/main/kotlin/com/malinskiy/marathon/android/executor/listeners/line/NullOutputListener.kt

```diff
@@ -0,0 +1,8 @@
+package com.malinskiy.marathon.android.executor.listeners.line
+
+/**
+ * Ignores the output
+ */
+class NullOutputListener : LineListener {
+    override fun onLine(line: String) {}
+}
```

Comment:
CMD + SHIFT + A **->** Ensure line feed at file end on Save option box (5 sec)

Reply:
Added .editorconfig to work around this